dbinit/src/main/liquibase/base/extra.yml

```diff
@@ -1,9 +0,0 @@
-databaseChangeLog:
```

Comment:
Nowhere. That's a mistake. I removed it when I tried to implement a different foreign key, but after that failed, I forgot to bring the unique constraint back.

I'll do it.